vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -176,13 +179,28 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
                 return existingPath;
             }
 
+            if(!(await WebRequestWorker.isOnline(timeoutValue ?? defaultTimeoutValue, globalEventStream)))
+            {
+                workerContext.acquisitionContext.architecture = workerContext.acquisitionContext.architecture ?? DotnetCoreAcquisitionWorker.defaultArchitecture();

```

Comment:
Can use `??=`

Reply:
omg thank you I forgot that operator existed 🤯